core/imageroot/usr/local/agent/bin/module-backup

```diff
@@ -64,6 +83,7 @@ except PermissionError:
 except Exception as ex:
     print("[ERROR] module-dump-state failed.", ex, file=sys.stderr)
     print("[ERROR] module-backup aborted.", file=sys.stderr)
+    errors += 1
```

Comment:
```suggestion
    backup['errors'] += 1
```

Reply:
I would prefer to stick to errors if you could accept